cypress-tests/README.md

```diff
@@ -1,30 +1,77 @@
-# Cypress Tests
+# Hyperswitch Cypress Testing Framework
 
 ## Overview
 
-This Tool is a solution designed to automate testing for the [Hyperswitch](https://github.com/juspay/hyperswitch/) using Cypress, an open-source tool capable of conducting API call tests and UI tests. This README provides guidance on installing Cypress and its dependencies.
-
-## Installation
-
-### Prerequisites
+This is a comprehensive testing framework built with [Cypress](https://cypress.io) to automate testing for [Hyperswitch](https://github.com/juspay/hyperswitch/). The framework supports API testing with features like multiple credential management, configuration management, global state handling, and extensive utility functions. The framework provides extensive support for API testing with advanced features including:
+
+- Multiple credential management
+- Dynamic configuration management
```

Comment:
Could you elaborate on what you mean here? 
Can first explain here on this GitHub PR thread, then we can decide on how to include it in the doc.

Reply:
- multiple credential support (mcs):
  - we already support multiple connectors. recently, we introduced this feature
  - mcs basically means a single connector can now support multiple api keys (`connector_1` and `connector_2`). max supported creds for a new connector cred is `2` at present (it was restricted to 2 recently due to compatibility issues with stripe's ntid requirements)
- dynamic configuration management:
  - we introduced a new field called as `Configs`. this is supposed to be passed in an exchange (with this, user, based on the need, can pass a configuration for a specific test and yes, this will affect everywhere for that connector. only 3 configs are supported at the moment of me writing this). for example, trustpay refunds, requires us to pass a `delay`, with this config, we can pass `delay` only to `refund`